libs/design/src/molecules/media-gallery/media-renderer/media-renderer.component.ts

```diff
@@ -0,0 +1,74 @@
+import {
+  Component,
+  OnInit,
+  ComponentFactoryResolver,
+  Input,
+  Type,
+  ViewChild,
+  ViewContainerRef,
+  TemplateRef,
+  ChangeDetectionStrategy,
+} from '@angular/core';
+
+import { DaffMediaGalleryComponent } from '../media-gallery.component';
+import { DaffMediaGalleryRegistry } from '../registry/media-gallery.registry';
+
+@Component({
+  selector: 'daff-media-renderer',
+  templateUrl: './media-renderer.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class DaffMediaRendererComponent implements OnInit {
+	@Input() component: Type<unknown>;
```

Comment:
Needs docs.

Reply:
added

Reply:
I'm not sure I like that the docs are written as "We..." It doesn't really match how we typically write docs?

Reply:
removed